src/main/java/dan200/computercraft/core/apis/HTTPAPI.java

```diff
@@ -76,90 +65,81 @@ public void shutdown( )
 
     @Nonnull
     @Override
-    public String[] getMethodNames()
-    {
-         return new String[] {
-            "request",
-            "checkURL"
+    public String[] getMethodNames() {
+        return new String[]{
+                "request",
+                "checkURL"
         };
     }
 
     @Override
-    public Object[] callMethod( @Nonnull ILuaContext context, int method, @Nonnull Object[] args ) throws LuaException
-    {
-        switch( method )
-        {
-            case 0:
-            {
+    public Object[] callMethod(@Nonnull ILuaContext context, int method, @Nonnull Object[] args) throws LuaException {
+        switch (method) {
+            case 0: {
                 // request
                 // Get URL
-                String urlString = getString( args, 0 );
+                String urlString = getString(args, 0);
 
                 // Get POST
-                String postString = optString( args, 1, null );
+                String postString = optString(args, 1, null);
 
                 // Get Headers
                 Map<String, String> headers = null;
-                Map<Object, Object> table = optTable( args, 2, null );
-                if( table != null )
-                {
-                    headers = new HashMap<>( table.size() );
-                    for( Object key : table.keySet() )
-                    {
-                        Object value = table.get( key );
-                        if( key instanceof String && value instanceof String )
-                        {
-                            headers.put( (String)key, (String)value );
+                Map<Object, Object> table = optTable(args, 2, null);
+                if (table != null) {
+                    headers = new HashMap<>(table.size());
+                    for (Object key : table.keySet()) {
+                        Object value = table.get(key);
+                        if (key instanceof String && value instanceof String) {
+                            headers.put((String) key, (String) value);
                         }
                     }
                 }
-                
+
                 // Get binary
                 boolean binary = false;
-                if( args.length >= 4 )
-                {
-                    binary = args[ 3 ] != null && !args[ 3 ].equals( Boolean.FALSE );
+                if (args.length >= 4) {
+                    binary = args[3] != null && !args[3].equals(Boolean.FALSE);
                 }
 
+                //Get mode
+                String DMODE = "GET";
```

Comment:
Also minor stylistic choice, but probably shouldn't be UPPER_CASE - that's normally reserved for constants.